python/grizzly/grizzly/grizzly_impl.py

```diff
@@ -213,6 +332,139 @@ def element_wise_op(array, other, op, ty):
                                           "ty": ty, "op": op}
     return weld_obj
 
+def unzip_columns(expr, column_types):
```

Comment:
What is the usecase of this method?

Reply:
Pulling apart a zipped expression.
This is mostly useful when materializing the columns of a dataframe. 
We can't materialize a vector of structs, and convert that to numpy arrays. So we unzip to have a struct of vectors which we can then convert to numpy arrays.